rest/route/github.go

```diff
@@ -645,6 +664,12 @@ func triggersPatch(action, comment string) (bool, string) {
 
 	return false, ""
 }
+func triggersStatusRefresh(action, comment string) bool {
+	if action == "deleted" {
+		return false
+	}
+	return comment == refreshStatusComment
```

Comment:
Can we please wrap this in a function that checks for whitespace, similar to how `isRetryComment` is set up?